src/js/components/Select/Select.js

```diff
@@ -255,7 +255,7 @@ const Select = forwardRef(
                   }
                   // When Select is disabled, we want to show a default cursor
                   // but not have disabled styling come from TextInput
-                  defaultCursor={disabled}
+                  defaultCursor={disabled === true || undefined}
```

Comment:
```suggestion
                  defaultCursor={disabled || undefined}
```
Should work as well

Reply:
disabled can also take an array that will disabled certain options. I think we only want to disable the TextInput if the control button should be disabled. this would occur only when disabled is true, rather than when an array of disabled options is defined.

Reply:
Good observation. would be helpful to add a comment there.

Reply:
sounds good, I will add it in